devices/frient/aqszb-110_voc_sensor.json

```diff
@@ -3,8 +3,8 @@
   "uuid": "14bd9844-c3e9-4831-a92d-91093da03de8",
   "manufacturername": "frient A/S",
   "modelid": "AQSZB-110",
-  "vendor": "Develco Products",
-  "product": "AQSZB-110 TVOC sensor",
```

Comment:
Is there any particular reason why you chop the vendor name here and add another vendor in the product name? Either you go with "Develco Products" or "frient" as vendor 🙂 
That goes for all frient devices.

Reply:
For now I have kept the vendor per directory but the DDF from the `develco` and `frient` directory should be merged, they are identical